app/assets/stylesheets/stats.scss

```diff
@@ -53,6 +53,10 @@
       content: image-url("stats_woman.png");
     }
 
+    &.other::before {
+      content: image-url("stats_other.png");
```

Comment:
@decabeza Do you know of an image we could use here, which is consistent with the images we use for men and women?